language_bindings/rust/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "halide"
+description ="Rust build script and runtime libaries for conviently useing Halide code in rust projects"
+version = "0.1.0"
```

Comment:
This will need to be kept in sync with the repository version somehow. Can this be programmed to parse the top-level CMakeLists.txt?